### ISSUE TITLE
Deprecated function in Gulp video

### ISSUE THREAD

**Issue by h-taylor** [joeeames/UnitTestingNodeCourse#1]
warning / error message I get when using the gulp run function: gulp.run() has been deprecated. Use task dependencies or gulp.watch task triggering instead.

Don't know if want stuff like this in the issues but here it is anyways.

**Comment by joeeames** [comment]
no, this is a great place for something like that. I am aware of the issue and looking at the appropriate fix

**Comment by michaeldperez** [comment]
Regarding the use of task dependencies, could one replace 

``` javascript
gulp.task('watch-mocha', function() {
  gulp.run('mocha');
  gulp.watch(['./**/*.js', 'test/**/*.js'], ['mocha']);
});
```

with:

``` javascript
gulp.task('watch-mocha', ['mocha'], function() {
  gulp.watch(['./**/*.js', 'test/**/*.js'], ['mocha']);
});
```

I believe this would get rid of the error @h-taylor mentions.
